internal/osutils/osutils.go

```diff
@@ -71,6 +78,24 @@ func winPathToLinPath(name string) (string, error) {
 	return path, nil
 }
 
+// BashifyPath takes a windows %PATH% list and turns it into a bash style PATH list.
+// e.g. C:\foo;C:\bar becomes /c/foo:/c/bar
+func BashifyPathEnv(pathList string) (string, error) {
+	if runtime.GOOS != "windows" {
+		return pathList, nil // already bashified
+	}
+
+	dirs := strings.Split(pathList, ";")
+	for i, dir := range dirs {
+		path, err := BashifyPath(dir)
```

Comment:
This will cause it to call `bash -c` in a loop for each path found, which will be slow and process intensive.

Might be worth researching some alternative solutions, but at the very least we could just resolve the first part of the path and apply the result of that to all matching path entries. In most cases that'd just be `C:\`.

Reply:
I've switched to doing a simple string translation. We only need it for bash to resolve executables in PATH, so we don't need fully resolved or simplified paths. The translation results in a valid path that can be read from disk.

Reply:
I switched back from doing a simple string translation. Now we leverage MinGW/MSYS2/WSL's auto-translation of %PATH%.